lib/network/modules/LayoutEngine.js

```diff
@@ -532,8 +543,8 @@ class LayoutEngine {
         let diffAbs = Math.abs(pos2 - pos1);
         //console.log("NOW CHEcKING:", node1.id, node2.id, diffAbs);
         if (diffAbs > this.options.hierarchical.nodeSpacing) {
-          let branchNodes1 = {}; branchNodes1[node1.id] = true;
```

Comment:
`getBranchNodes` now requires an empty map rather than one in which the first node is populated; since `getBranchNodes` always populates the map for the first node, this was redundant anyway.